### PR TITLE
add active class in summary menu for custom tabs

### PR DESCRIPTION
**Pull request by shaozi**
This is to fix issue [23418](https://mantisbt.org/bugs/view.php?id=23418)

### CODE REVIEW

File: core/html_api.php [hunk 1]
```diff
@@ -894,7 +894,10 @@ function print_summary_menu( $p_page = '', array $p_filter = null ) {
 
 	# Plugins menu items - these are cooked links
 	foreach ( $t_menu_options as $t_menu_item ) {
-		echo '<li>' . $t_menu_item . '</li>';
+		$t_active = $p_page && strpos( $t_menu_item, $p_page ) !== false
+			? ' class="active"'
+			: '';
+		echo "<li{$t_active}>" . $t_menu_item . '</li>';
```
Comment (dregad) [inline core/html_api.php]: Since there's already a substitution, might as well do this instead

    echo "<li{$t_active}>{$t_menu_item}</li>";

Comment (shaozi) [reply core/html_api.php]: just did a commit

Comment (dregad) [reply core/html_api.php]: That's perfect, thanks

**Comment by vboctor** [comment]
Thanks @shaozi for your contribution.  Merged via 113a270b69b0299a240c746c525467ec21689bed

**Pull request closed without merging**